BGL/include/CGAL/boost/graph/copy_face_graph.h

```diff
@@ -208,7 +212,39 @@ void copy_face_graph(const SourceMesh& sm, TargetMesh& tm,
                        sm_vpm, tm_vpm);
 }
 
+
+
 } // end of namespace internal
+namespace impl
```

Comment:
The idea of having another namespace is to avoid conflicts like those #3092 are trying to fix. `impl` to too generic IMO but we can keep it and fix that when needed.

Reply:
I can call it Gerard if you prefer ;)